documentation/modules/security/proc-installing-certs-per-listener.adoc

```diff
@@ -3,21 +3,41 @@
 // assembly-security.adoc
 
 [id='proc-installing-certs-per-listener-{context}']
-= Providing your own Kafka listener certificates
+= Providing your own Kafka listener certificates for TLS authentication
 
-This procedure shows how to configure a listener to use your own private key and server certificate, called a xref:kafka-listener-certificates-{context}[Kafka listener certificate].
+[role="_abstract"]
+Listeners provide client access to Kafka brokers.
+You configure listeners in the `Kafka` resource, including the configuration required for client access using TLS authentication.
 
-Your client applications should use the CA public key as a trusted certificate in order to verify the identity of the Kafka broker.
+By default, the listeners use certificates signed by the internal self-signed CAs generated by Strimzi.
+A public CA certificate is generated by the Cluster Operator when it creates a Kafka cluster.
+When you configure a client for TLS, you add the public CA certificate to its truststore configuration to verify the Kafka cluster.
+You can also xref:installing-your-own-ca-certificates-str[install and use your own CA certificates]. 
+Or you can configure a listener using `brokerCertChainAndKey` properties and use a custom server certificate.
+
+The `brokerCertChainAndKey` properties allow you to authenticate access to Kafka brokers using your own custom certificates at the listener-level.
+You create a secret with your own private key and server certificate, then specify the key and certificate in the listener's `brokerCertChainAndKey` configuration.
+The server certificate must be signed by a CA.
```

Comment:
It does not have to be signed by a CA. But if it would not be, there is no sense in using the custom certificate. So I would remove this.